### ISSUE TITLE
Display all one-shots

### ISSUE THREAD

**Issue by scooper4711** [frondeus/fvtt-syrin-control#12]
I'd like to be able to see all one-shots, not just the ones local to the scene. I need my Wilhelm Scream at the drop of a hat.
(Yeah, I'm going to make a macro for it. thanks for that)

**Comment by frondeus** [comment]
Hi!

Do you use the CSV synchronization method?
Unfortunately, last time I checked the CSV did not include global one-shots.

The API method does load global one-shots.

For now there are two steps to do:
* I need to investigate the CSV content again to see if its really missing globals.
* If yes, then I should write a guide how to create global macro anyway. It is possible but its inconvenient:
For now, shortened version:
* You need to go to Master Panel, and click "Show Remote Control links".
Now you can click your Wilhelm Scream, it should give you a link like:
```
https://syrinscape.com/online/frontend-api/elements/2357/play/?auth_token=<your_token>
```
Next, you copy the ID, in that case, `2357`.
And paste it in your macro:
```
game.syrinscape.playElement(2357)
```

**Comment by frondeus** [comment]
Unfortunately, I checked CSV, and while there is Wilhelm scream, it's one linked to mood, not a global one.

**Comment by frondeus** [comment]
Closed: 0.3.0 uses API which fetches global one-shots properly.
